examples/memo/index.js

```diff
@@ -0,0 +1,82 @@
+"use strict";
+
+// import base modules
+let cottage = require("cottage");
+let bodyParser = require('koa-bodyparser');
+
+// import setting, database module
+let setting = require("./setting.json");
+let database = require("./database.js");
+let Memo = database.model("Memo");
+let ObjectId = database.Types.ObjectId;
+
+// using bodyparser and static serving
+let app = cottage({ caseSensitive: true });
+app.use(bodyParser());
+
+function findProperties (array, object) {
+	var returnVal = true;
+	for (let val of array) if (!object.hasOwnProperty(val)) returnVal = false;
+	return returnVal;
+}
+
+app.get("/memo", function *(req, res) {
+	// Check validation
+	if (!findProperties(["length", "pageNo"], req.body)) {
+    res.status = 412;
+    return;
+  }
+
+  let length = parseInt(req.body.length);
+	let offset = (parseInt(req.body.pageNo) - 1) * length;
+	return yield Memo.find({}, null, { skip: offset, limit: length });
+});
+
+app.get("/memo/:memoId", function *(req, res) {
+	return yield Memo.findById(new ObjectId(req.params.memoId));
```

Comment:
`new ObjectId` is redundant because mongo converts `string` to `ObjectId` automatically